src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -2514,6 +2570,22 @@ public function isEnabled(): bool
     }
 }
 
+#[AsCommand(name: 'invokable')]
```

Comment:
```suggestion
#[AsCommand(name: 'invokable', aliases: 'invk')]
```

I did some testing on Drush and I am seeing the PR fail when an invokable command uses an alias. If you accept my suggestion above, hopefully the tests fail. 

The problem is that `$attribute->name` is a pipe delimited string consisting of the name, followed by the aliases. See `\Symfony\Component\Console\Attribute\AsCommand::__construct`. 

The code below helped me get past this bug. Not sure if there are better ways to solve it

```
$aliases = explode('|', $attribute->name);
$name = array_shift($aliases);
$commandHandler = (new Command($name))
    ->setAliases($aliases)
    ...
       
```

Reply:
See #60739